### PR TITLE
[03-matrix-multiplication] Enable commented out configs

### PR DESCRIPTION
**Pull request by whitneywhtsang**

### CODE REVIEW

**Review by etiotto — COMMENTED** [review]
LGTM

File: python/tutorials/03-matrix-multiplication.py [hunk 1]
```diff
@@ -170,11 +170,15 @@ def is_xpu():
     return triton.runtime.driver.active.get_current_target().backend == "xpu"
 
 
+def get_xpu_autotune_config():
```
Comment (etiotto) [inline python/tutorials/03-matrix-multiplication.py:173]: Hmmm... so the idea is to use a different set of ` triton.Config` here ? Why the configurations used by NVidia aren't good ?

Comment (whitneywhtsang) [reply python/tutorials/03-matrix-multiplication.py:173]: You can see that there are also configs specifically for amd in `get_hip_autotune_config`, different GPU should autotune for different configs, e.g., in general `num_warps` should be bigger for xpu compare to cuda.

Comment (etiotto) [reply python/tutorials/03-matrix-multiplication.py:173]: I can see yes. We should fill this part with config that are customized for Intel asap

**Pull request merged**